src/python/pants/notes/1.21.x.rst

```diff
@@ -0,0 +1,214 @@
+1.21.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.21.x`` ``stable`` series.
+
+1.21.0rc0 (09/23/2019)
+----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add "did you mean" functionality to runtime Param mismatches. (#8315)
+  `PR #8315 <https://github.com/pantsbuild/pants/pull/8315>`_
+
+Bugfixes
+~~~~~~~~
+* Fix support for running pants from source in other repositories. (#8297)
+  `PR #8297 <https://github.com/pantsbuild/pants/pull/8297>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Upgrade to Pex 1.6.11. (#8287)
+  `PR #8287 <https://github.com/pantsbuild/pants/pull/8287>`_
+
+* Implement critical path algorithm in Graph (#8308)
+  `PR #8308 <https://github.com/pantsbuild/pants/pull/8308>`_
+
+* Bump to latest zinc and remove the extractor. (#8246)
+  `PR #8246 <https://github.com/pantsbuild/pants/pull/8246>`_
+
+* Skip test_ctypes_third_party_integration due to network failure. (#8317)
+  `PR #8317 <https://github.com/pantsbuild/pants/pull/8317>`_
+
+* repr() exceptions when logging/stringifying them. (#8230)
+  `PR #8230 <https://github.com/pantsbuild/pants/pull/8230>`_
+
+* On stats v2, use json body, API will support uploading multiple builds in a single call (#8305)
+  `PR #8305 <https://github.com/pantsbuild/pants/pull/8305>`_
+
+* Hide zinc-compiler `BareBonesLogger` behind flag (#8295)
+  `PR #8295 <https://github.com/pantsbuild/pants/pull/8295>`_
+  `PR #8279#pullrequestreview-288379413) <https://github.com/pantsbuild/pants/pull/8279#pullrequestreview-288379413)>`_
```

Comment:
This link text could probably be shortened to \`PR #8279 <...>\`_  or the link could be dropped.